app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/manage_your_details.scala.html

```diff
@@ -76,7 +76,7 @@ <h2 class="heading-medium">
   @cyaRow(
     messages("subscription.cyaLabel.email"),
     Html(details.emailAddress.value),
-    Some(cyaChange(messages("subscription.cyaChange.email"), "#"))
+    Some(cyaChange(messages("subscription.cyaChange.email"), "/cgt-property-disposals/subscribed/amend-details/change-email")))
```

Comment:
you can use the reverse routing stuff here to be able to cope better with possible future changes to the url, so you could have `email.routes.BlahController.blah` here